### ISSUE TITLE
StackOverflowError on Tutorial.toString()

### ISSUE THREAD

**Issue by Brutus5000** [FAForever/faf-java-commons#18]
Looks like a circular reference of the relationships. Probably just needs some refinement in Lomboks @ToString.

Happens on deleting a tutorial in faf-moderator-client.
User actions are currently not affected.

Shortcut of the Stacktrace:
```
SLF4J: Failed toString() invocation on an object of type [com.faforever.commons.api.dto.Tutorial]
Reported exception:
java.lang.StackOverflowError
	at java.util.AbstractCollection.toString(AbstractCollection.java:454)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at com.faforever.commons.api.dto.TutorialCategory.toString(TutorialCategory.java:12)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at com.faforever.commons.api.dto.Tutorial.toString(Tutorial.java:7)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
```

**Comment by basalt79** [comment]
I will have a look

**Comment by Brutus5000** [comment]
Fixed by  fb35c5d
